src/patternfly/components/Dropdown/dropdown-menu-toggle-image-text.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="pf-c-dropdown__menu{{#if dropdown-menu--modifier}} {{dropdown-menu--modifier}}{{/if}}"
+  aria-labelledby="{{id}}-button"
+  {{#unless dropdown--IsExpanded}}hidden{{/unless}}
+  {{#if dropdown-menu--attribute}}
+    {{{dropdown-menu--attribute}}}
+  {{/if}}>
+
+  {{#> dropdown-group}}
+    {{#> dropdown-menu-item dropdown-menu-item--type="div" dropdown-menu-item--modifier="pf-m-text"}}
+      Foo text
```

Comment:
Can you update this so it's something like "Text" and the second one is "More text", or something like that?